packages/astro/src/assets/fonts/vite-plugin-fonts.ts

```diff
@@ -272,16 +278,14 @@ export function fontsPlugin({ settings, sync, logger }: Options): Plugin {
 				const { cached, data } = await cache!(hash, () => fetchFont(url));
 				logManager.remove(hash, cached);
 
-				// TODO: add cache control back
-				// TODO: set content type and cache control manually
-				// const keys = ['cache-control', 'content-type', 'content-length'];
-				// const keys = ['content-type', 'content-length'];
-				// for (const key of keys) {
-				// 	const value = response.headers.get(key);
-				// 	if (value) {
-				// 		res.setHeader(key, value);
-				// 	}
-				// }
+				res.setHeader('Content-Length', data.length);
+				res.setHeader('Content-Type', `font/${extractFontType(hash)}`);
+				// We don't want the request to be cached in dev because we cache it already internally,
+				// and it makes it easier to debug without needing hard refreshes
+				res.setHeader('Cache-Control', 'no-store, no-cache, must-revalidate, max-age=0');
+				res.setHeader('Pragma', 'no-cache');
+				res.setHeader('Expires', 0);
```

Comment:
Do you think this information should be part of the roadmap, and eventually documented? I believe users might expect fonts to be browser-cached, or they would expect production to behave like dev.

Reply:
Yes that's a good point, I'm going to update it